README.md

```diff
@@ -68,6 +68,14 @@ To do your calls with SNI, pass `:with-sni? true` to your calls.
 For example,
 `(mauth/get! "https://www.mdsol.com" "/api/v2/testing.json" :with-sni? true)`
 
+Version 2.0.1 update -
+We have added support for mauth version 2 headers as well now. To get the v2 headers you need to add additional header in your request as
+"mauth-version" with value as "v2" and algorithm also uses query param if any to sign  or to generate these headers, so you will need to add additional header in your request as
+"query-param-string"  and value should be string of sorted and encoded query params e.g abc=abctest&test=testing%20value
```

Comment:
Can you explain why it is designed this way? Please let's connect cc: @rbhanushali @priyankaMedidata

Reply:
as discussed in demo marking this as resolved